firstsection.css

```diff
@@ -0,0 +1,542 @@
+.first {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.content {
+    overflow-x: hidden;
+    overflow-y: visible;
+}
+.content__banner {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    position: relative;
+    margin-bottom: 43px;
+    text-transform: uppercase;
+    color: white;
+}
+.banner {
+    width: 100%;
+}
+.banner__img {
+    object-fit: cover;
+    object-position: center;
+    width: 100%;
+}
+.banner__title {
+    font: bold 30px/29px 'Gotham Pro';
+    position: absolute;
+    top: 76px;
+    left: 54px;
+    width: 209px;
+    margin: 0;
+}
+.banner__shop {
+    font: bold 12px 'Gotham Pro';
+    position: absolute;
+    top: 157px;
+    left: 54px;
+    width: 129px;
+    margin: 13.5px 0 0 0;
+}
+.banner__shop_link {
+    display: inline-block;
+    color: white;
+}
+.link__text {
+    margin: 0 7px 0 0;
+}
+.link-arrow {
+    width: 46px;
+}
+@media screen and (min-width: 1025px) {
+    .banner__title {
+        top: 91px;
+        left: 60px;
+        width: 195px;
+    }
+    .content__banner {
+        margin-bottom: 30px;
+    }
+}
+@media screen and (min-width: 1025px) {
+    .banner__shop {
+        top: 177px;
+        left: 60px;
+        margin: 15px 0 0 0;
+    }
+}
+@media screen and (min-width: 1025px) {
+    .link__text {
+        margin: 0 3.5px 0 0;
+    }
+}
+@media screen and (min-width: 1025px) {
+    .link-arrow {
+        width: 43px;
+    }
+}
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+    .content__banner {
+        margin-bottom: 36px;
+    }
+    .banner__shop {
+        top: 273px;
+        left: 43px;
+        width: 210px;
+        font: bold 20px 'Gotham Pro';
+        margin: 36.5px 0 0 0;
+    }
+    .banner__title {
+        width: 355px;
+        font: bold 50px/48px 'Gotham Pro';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)